lib/config/defaults.js

```diff
@@ -704,16 +704,24 @@ const applyModuleDefaults = (
 			});
 		}
 		if (css) {
+			const styleResolutionDefaults = {
+				conditionNames: ["style"],
+				mainFields: ["css", "style", "main", "..."],
```

Comment:
I don't see `css` field never, can you provide an example? Better to avoid example logic here, because other bundlers will need to do it

Reply:
Also let's add test cases:
- `main` has `css` file, we should resolve it, it is very legacy, but still can be found
- we should support `byDependency`, so I think it is a wrong place, we should put it here https://github.com/webpack/webpack/blob/15e1e2a18b31f9781a467a5a6c995ed096a8bb2b/lib/config/defaults.js#L1457 and then use here the same https://github.com/webpack/webpack/blob/15e1e2a18b31f9781a467a5a6c995ed096a8bb2b/lib/config/defaults.js#L605, but only for CSS, currently we use `css-import` keyword, but maybe we need to rename it to just `css`
- Will be great if you look at `css-loader` and found all tests related to import from `node_modules` and move them here

Reply:
Used the existing config from style loader to keep it consistent - https://github.com/webpack-contrib/css-loader/blob/master/src/plugins/postcss-import-parser.js#L231

Reply:
You are right. Removing support for css

Reply:
Will add tests from style-loader shortly